src/ethereum_spec_tools/evm_tools/t8n/evm_trace.py

```diff
@@ -125,7 +147,11 @@ def evm_trace(
     len_memory = len(evm.memory)
 
     return_data = None
-    if trace_return_data and evm.return_data:
+    if (
+        isinstance(evm, EvmWithReturnData)
+        and trace_return_data
+        and evm.return_data
```

Comment:
Should empty return data on forks with return data be `"0x"` or `None`? This implements `None`, `"0x"` seems more logical to me.

Reply:
Updated